### DISCUSSION TITLE
Building and installing fpm with installed fpm

### DISCUSSION THREAD

**Discussion by nncarlson** [fortran-lang/fpm#668] [General]
I cloned the fpm repo and did the `./install.sh` thing to bootstrap to get an installed fpm.  I then used that fpm to build my checked out copy of the fpm repo.  That worked fine.  But `fpm install` failed with "text file busy" when it tried to copy the newly built binary over the installed binary which I was using.  I realize I can just manually copy things after the fact, but my question is whether this should have worked.

**Comment by freevryheid** [comment]
yep, this is on linux. Not sure how it behaves on windows. The install code tries to cp the fpm executable but if the file already exists it should rather mv or rm first to unlink.

**Comment by freevryheid** [comment]
Just a heads up that a fix for this was recently merged.
